site/public/thrift/zipkinDependencies.thrift

```diff
@@ -0,0 +1,39 @@
+#
```

Comment:
seems like this is still copying IDL (thrift, openapi and proto) from the prior location. By flattening I meant to make this the root project (so no site). Does that help avoid copying or moving these files from their current location?

Reply:
I'm not sure if this is beneficial. If I'd like to check out the yaml myself I would had to know that this is hidden in a subfolder under side.

Reply:
this won't merge with copies of files in the same repo as it subverts the purpose of git.